spec/models/rdv/send_webhook_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe Rdv, type: :model do
+  include ActiveJob::TestHelper
+
+  before(:each) do
+    create(:webhook_endpoint)
+  end
+
+  after(:each) do
+    clear_enqueued_jobs
+  end
+
+  describe '#send_web_hook' do
+    let(:rdv) { create(:rdv) }
+
+    describe 'default publication' do
+      it 'publishes the default status' do
+        expect(WebhookJob).to receive(:perform_later).with(hash_including('status' => 'unknown'), instance_of(WebhookEndpoint))
+        rdv.reload
+      end
+    end
+
+    describe 'publication on update' do
```

Comment:
I'm not sure splitting into 'describe' blocks is necessary here, but if you do : 
- use `context` instead of describe. they do exactly the same (that is, nothing), but semantically describe is used with method names or behaviour names whereas context is used with, well, specific context cases
- the destroy example should not be in the update context